sql/moz-fx-data-shared-prod/telemetry_dev_cycle_external/experiments_stats_v1/query.py

```diff
@@ -10,7 +10,9 @@
 from google.cloud import bigquery
 
 API_BASE_URL_EXPERIMENTS = "https://experimenter.services.mozilla.com"
-API_BASE_URL_METRIC_HUB = "https://github.com/mozilla/metric-hub/tree/main/jetstream"
+API_BASE_URL_METRIC_HUB = (
+    "https://api.github.com/repos/mozilla/metric-hub/contents/jetstream"
```

Comment:
Flyby comment: maybe we should version this API to avoid this in the future (idk who to cc here)